src/test/encoding/types.h

```diff
@@ -114,6 +114,7 @@ TYPE(ObjectStore::Transaction)
 #include "os/filestore/SequencerPosition.h"
 TYPE(SequencerPosition)
 
+#if defined(HAVE_LIBAIO)
```

Comment:
could replace the "FreeBSD/Bluestore" prefix in the title of commit message with "test/encoding/types.h". because bluestore does not compile without libaio.